pkg/cmd/attestation/verify/verify.go

```diff
@@ -52,7 +52,7 @@ func NewVerifyCmd(f *cmdutil.Factory, runF func(*Options) error) *cobra.Command
 			provide a path to the %[1]s--bundle%[1]s flag.
 
 			To see the full results that are generated upon successful verification, i.e.
-			for use with a policy engine, provide the %[1]s--json-result%[1]s flag.
+			for use with a policy engine, provide the %[1]s--format=json%[1]s flag.
```

Comment:
Checks out 🤔 assuming `cmdutil.AddFormatFlag()`

https://github.com/cli/cli/blob/e8a13cfed3153301335b5afa18df25708db404fb/pkg/cmd/attestation/verify/verify.go#L147

https://github.com/cli/cli/blob/31a9d083bd0feaf237e7cf33b6171a1e902b7d3a/pkg/cmdutil/json_flags.go#L122-L148

Reply:
I'd love to be able to customize the `StringP` but that can wait for a future PR.